DiceKit/FrequencyDistribution.swift

```diff
@@ -198,6 +202,11 @@ extension FrequencyDistribution {
         return mapFrequencies { $0 / frequencies }
     }
     
+    public func filterZeroFrequencies(delta: Frequency) -> FrequencyDistribution {
+        let newFrequenciesPerOutcome = frequenciesPerOutcome.filterValues {abs($0) > delta}
```

Comment:
I don't like that this and the initializer are slightly different. I feel that they should use the same code path.